experimental/Schemes/elliptic_surface.jl

```diff
@@ -2119,7 +2136,71 @@ function morphism_from_section(X::EllipticSurface, i::Int)
   Fkkt = fraction_field(kkt)
   img_gens2 = Fkkt.(img_gens)
   # TODO: Cache?
-  return morphism_from_rational_functions(B, X, V, U, img_gens2, check=false)
+  iso = morphism_from_rational_functions(B, C, V, UC, img_gens2, check=false)
+  return iso, inc_C
+end
+
+########################################################################
+# Translations by sections                                             #
+########################################################################
+
+function translation_morphism(X::EllipticSurface, P::EllipticCurvePoint;
+    divisor::AbsWeilDivisor=_section(X, P)
+  )
+  E = generic_fiber(X)
+  @assert parent(P) === E "point does not lay on the underlying elliptic curve"
+  U = weierstrass_chart_on_minimal_model(X)
+  is_zero(P) && return identity_map(X)
+
+  # We construct the translation by P as a morphism of rational functions
+  kT = base_field(E)
+  T = first(gens(kT))
+
+  R = ambient_coordinate_ring(U)
+  x, y, t = gens(R)
+  
+  a1, a2, a3, a4, a6 = [evaluate(a, t) for a in a_invariants(E)]
+
+  p_x = evaluate(P[1], t)
+  p_y = evaluate(P[2], t)
+
+  # Formulas adapted from Hecke/src/EllCrv/EllCrv.jl
+  m = (p_y - y)//(p_x - x)
+  pb_x = - x - p_x - a2 + a1*m + m^2
+  pb_y = - y - m*(pb_x - x) - a1*pb_x - a3
+
+  F = fraction_field(R)
+
+  result = morphism_from_rational_functions(X, X, U, U, F.([pb_x, pb_y, t]), check=true)
+  set_attribute!(result, :is_isomorphism=>true)
+  return result
+end
+
+function _pushforward_section(
+    phi::MorphismFromRationalFunctions{<:EllipticSurface, <:EllipticSurface}, 
+    P::EllipticCurvePoint;
+    divisor::AbsWeilDivisor=_section(domain(phi), P),
+    codomain_charts::Vector{<:AbsAffineScheme} = affine_charts(codomain(phi))
+  )
```

Comment:
We should use this procedure here in general for sections. Today it seemed that `simplify` can not really take care of finding the parametrization.